framework/core/src/Foundation/ApplicationInfoProvider.php

```diff
@@ -119,7 +119,7 @@ public function getSchedulerStatus(): string
         // If the schedule has not run in the last 5 minutes, mark it as inactive.
         return Carbon::parse($status) > Carbon::now()->subMinutes(5)
             ? $this->translator->trans('core.admin.dashboard.status.scheduler.active')
-            : $this->translator->trans('core.admin.status.scheduler.inactive');
```

Comment:
It seems to be the other way around?

https://github.com/flarum/framework/blob/d8fa791d9c4077670941ac5cd0203a791fb3ec50/framework/core/locale/core.yml#L55-L63

`dashboard` should be `status`?

Reply:
Never mind I failed seeing the addition of the word. 👍